lib/machete/matchers/literal_map_matcher.ex

```diff
@@ -43,8 +41,8 @@ defmodule Machete.LiteralMapMatcher do
 
       missing_keys =
         missing_keys
-        |> MapSet.difference(atom_keys |> Enum.map(&Kernel.to_string/1) |> MapSet.new())
-        |> MapSet.difference(string_keys |> Enum.map(&String.to_existing_atom/1) |> MapSet.new())
+        |> MapSet.difference(MapSet.new(atom_keys, &Kernel.to_string/1))
+        |> MapSet.difference(MapSet.new(string_keys, &safe_existing_atom/1))
```

Comment:
The diff on the first line here is not strictly necessary for the fix, but I figured I'd save an unnecessary list creation where possible.